src/config.js

```diff
@@ -261,6 +261,14 @@ export const config = {
                 return "https://cmssdt.cern.ch/SDT/cgi-bin/igprof-navigator/" + getCurrentIbTag(ib);
             }
         },
+        {
+            //add_igprof_tests_link
+            key: "vtune",
+            name: "Vtune",
+            getUrl: function (ib) {
+                return "https://cmssdt.cern.ch/vtune/ui/" + getCurrentIbTag(ib);
```

Comment:
Direct link requires IB directory and file name.

Reply:
This would work if I can pass in arch and workflow number
https://cmssdt.cern.ch/vtune/ui/CMSSW_14_1_X_2024-05-07-1100/el8_amd64_gcc12/23834.21/r-step3-23834.21-hs
or I can just remove `getCurrentIbTag(ib)`

Reply:
I added a default workflow and filename..